client/src/components/ResetButton.tsx

```diff
@@ -0,0 +1,27 @@
+import {useAtom} from 'jotai';
+import {resetApp} from 'lib/api/reset';
+import React from 'react';
+import {datasetsAtom, modelsAtom, transcriptionsAtom} from 'store';
+
+export default function ResetButton() {
+  const [, setDatasets] = useAtom(datasetsAtom);
+  const [, setModels] = useAtom(modelsAtom);
+  const [, setTranscriptions] = useAtom(transcriptionsAtom);
+
+  const reset = async () => {
+    const response = await resetApp();
+    if (response.ok) {
+      setDatasets([]);
+      setModels([]);
+      setTranscriptions([]);
+    } else {
+      console.error('Error resetting app');
```

Comment:
Would it be worth showing the error message to help diagnose if the reset failed?

Reply:
I can make this an alert if you'd prefer for all users to see this (which I don't think any will be still). For context, the only way the request is going to fail here should be if the server container is down. The server container also get's restarted immediately if it ever goes down.

Reply:
Ah, that's fine then. No need to propagate that error.